utils/Makefile

```diff
@@ -0,0 +1,20 @@
+CXX := g++
+LDFLAGS := -lclickhouse-cpp-lib
```

Comment:
Do we really need to build ClickHouse server from sources?
Maybe there is a mix-up in the projects? The user space utility requires https://github.com/ClickHouse/clickhouse-cpp.git.

Reply:
My build fails without  -lzstd -llz4 -lcityhash

Reply:
Need to remember how I resolved it. I think, I compiled clickhouse-cpp with static libs, but it looks, it would be better to compile dynamically and add  -lzstd -llz4 -lcityhash to the Makefile.

Reply:
Did you install libabsl-dev?
On Ubuntu 22 I had such dependencies:
cmake
ninja-build
libabsl-dev
libboost-program-options1.74-dev
libfmt-dev
libspdlog-dev